content/docs/dvclive/get-started.md

```diff
@@ -1,134 +1,129 @@
-live.log_image("image.png", img)
+```dvc
+$ dvc init
 ```
 
-See `Live.log_image()`.
+After that, we use `dvc exp init --live` to create a `dvc.yaml` file:
 
-</tab>
-<tab title="Plots">
-
-```python
-y_true = [0, 0, 1, 1]
-y_pred = [0.2, 0.5, 0.3, 0.8]
-live.log_plot("roc", y_true, y_score)
-```
-
-See `Live.log_plot()`.
+<admon type="info">
 
-</tab>
-</toggle>
+Note that the path passed to the `--live` option (`"training_metrics"`) matches
+the `path` passed to the callback in the Python code.
 
-### (Optionally) Update the step number
+</admon>
 
-```python
-live.next_step()
+```dvc
+$ dvc exp init \
+--live "training_metrics" \
+--code "train.py" \
+python train.py
 ```
 
-See `Live.next_step()` and `Live.set_step()` for details.
-
-## Putting all together
-
-Joining the above snippets, you can include DVCLive in your training code:
-
-```python
-# train.py
-
-from dvclive import Live
-
-live = Live()
+<details>
+
+### ⚙️ Expand to see the `dvc.yaml`
+
+The `dvc.yaml` will contain a new `train` stage using the DVCLive outputs as
+`metrics` and `plots`:
+
+```yaml
+stages:
+  train:
+    cmd: python train.py
+    deps:
+      - train.py
+    metrics:
+      - training_metrics.json:
+          cache: false
+    plots:
+      - training_metrics/scalars:
+          cache: false
+```
 
-for epoch in range(NUM_EPOCHS):
-    train_model(...)
-    metrics = evaluate_model(...)
+You can find more details about the structure of files generated by DVCLive in
+the [API Reference](/doc/dvclive/api-reference#outputs)
 
-    for metric_name, value in metrics.items():
-        live.log(metric_name, value)
+</details>
 
-    live.next_step()
-```
+## Run experiments
 
-## Outputs
-
-After you run your training code, you should see the following content in the
-project:
+You can now
+[run experiments](/doc/user-guide/experiment-management/running-experiments)
+with `dvc exp run`:
 
 ```dvc
-$ tree
-├── dvclive
-│   ├── images
-│   │   └── img.png
-│   ├── plots
-│   │   └── roc.json
-│   └── scalars
-│       └── acc.tsv
-└── dvclive.json
+$ dvc exp run
 ```
 
-The contents of the `dvclive` folder and `dvclive.json` will vary depending on
-the type of data you have logged and whether you have updated the step value or
-not.
+## Compare and visualize experiments
 
-See `Live.log()`, `Live.log_image()` and `Live.log_plot()` for more details.
+After following the above steps we have enabled different ways of monitoring the
+training, comparing, and visualizing the experiments:
 
-### Metrics report
+### DVC
 
-If and when `step` is updated, DVCLive generates or updates a report in
-`dvclive/report.{format}` which will contain all the logged data.
+After the experiment has finished, you can use `dvc exp show` / `dvc plots diff`
+to compare and visualize experiments.
 
-![](/img/dvclive-html.gif)
+While the experiment is running, you can monitor the training with the
+[metrics report](/docs/dvclive/api-reference#metrics-report) that DVCLive
```

Comment:
Again, I do like this approach a bit more but I feel like it's lacking at least mentioning the vanilla use case (that DVCLive generates a report even if you don't use DVC).

Reply:
Yeah, I'd like to see this at the top before you get to DVC. Show that you can run the code normally and get the live metrics report, then move on to DVC.

Reply:
It has the added benefit of faster time to value: show something useful (the report) almost immediately.

Reply:
"faster time to value" is the motto of [Get Started type docs](https://www.notion.so/iterative/Higher-level-content-9408f84a9b694fab949feede39142122#8e6381fcf06643aaa1d8df100fd4014a) 🤩